### PR TITLE
Added Wraith templates and files for D7 and D8.

### PR DESCRIPTION
**Pull request by dharizza**
Acceptance criteria:
- [x] Projects generated in both, D7 and D8, has wraith files.
- [x] Following the steps defined in README.md makes possible to execute visual regression tests.

### CODE REVIEW

File: generators/app/templates/7/_README.md [hunk 1]
```diff
@@ -25,6 +25,39 @@ vagrant plugin install vagrant-hostsupdater
...
+In this generator we include Wraith for visual regression testing so you'll need the following in order to be able to use it:
+
+* Ruby (it has been tested with Ruby 2.4).
+* Imagemagick
+* PhantomJS
+
+To install those you can run:
+
+```bash
+sudo apt-get install libicu-dev imagemagick rake phantomjs
```
Comment (kporras07) [inline generators/app/templates/7/_README.md:38]: @dharizza Could you please try to add instructions for mac?

File: generators/app/templates/7/_README.md [hunk 2]
```diff
@@ -25,6 +25,39 @@ vagrant plugin install vagrant-hostsupdater
...
+```
+
+And you can also install casperjs:
+
+```bash
+sudo npm install -g casperjs
```
Comment (kporras07) [inline generators/app/templates/7/_README.md]: Please remove sudo

File: generators/app/templates/7/_wercker.yml [hunk 3]
```diff
@@ -63,6 +63,17 @@ build:
             drush master-execute -y
             drush cc all
 
+            # Install dependencies for running visual regression tests with Wraith.
+            \curl -sSL https://get.rvm.io | bash -s stable
+            \curl -sSL https://get.rvm.io | bash -s -- --ignore-dotfiles
+            source $HOME/.rvm/scripts/rvm
+            rvm install 2.4
+            rvm use 2.4
+            ruby -v
```
Comment (kporras07) [inline generators/app/templates/7/_wercker.yml]: This is supposed to be debug info; you can safely remove it.

File: generators/app/templates/8/_README.md [hunk 4]
```diff
@@ -25,6 +25,40 @@ vagrant plugin install vagrant-hostsupdater
...
+In this generator we include Wraith for visual regression testing so you'll need the following in order to be able to use it:
+
+* Ruby (it has been tested with Ruby 2.4).
+* Imagemagick
+* PhantomJS
+
+To install those you can run:
+
+```bash
+sudo apt-get install libicu-dev imagemagick rake phantomjs
```
Comment (kporras07) [inline generators/app/templates/8/_README.md:38]: Same here

File: generators/app/templates/8/_README.md [hunk 5]
```diff
@@ -25,6 +25,40 @@ vagrant plugin install vagrant-hostsupdater
...
+```
+
+And you can also install casperjs:
+
+```bash
+sudo npm install -g casperjs
```
Comment (kporras07) [inline generators/app/templates/8/_README.md]: and here

File: generators/app/templates/8/_wercker.yml [hunk 6]
```diff
@@ -61,6 +61,17 @@ build:
             drush cc drush
             drush cr
 
+            # Install dependencies for running visual regression tests with Wraith.
+            \curl -sSL https://get.rvm.io | bash -s stable
+            \curl -sSL https://get.rvm.io | bash -s -- --ignore-dotfiles
+            source $HOME/.rvm/scripts/rvm
+            rvm install 2.4
+            rvm use 2.4
+            ruby -v
```
Comment (kporras07) [inline generators/app/templates/8/_wercker.yml]: and here

**Review by kporras07 — COMMENTED** [review]
It looks nice! I left you a couple of comments :)

**Comment by dharizza** [comment]
Ok, thanks, I've just fixed those.

**Comment by kporras07** [comment]
@dharizza :shipit:

**Pull request merged**